DESCRIPTION

```diff
@@ -9,4 +9,11 @@ Depends: R (>= 3.0.0)
 Description: Tools for working with Gustavo Niemeyer's geohash coordinate system, including API for interacting with other common R GIS libraries.
 URL: https://github.com/MichaelChirico/geohashTools
 License: MPL-2.0 | file LICENSE
-Suggests: rgdal, sf, sp, testthat
+Suggests: 
+    rgdal,
+    sf,
+    sp,
+    testthat,
+    knitr,
+    rmarkdown
```

Comment:
From [WRE](https://cran.r-project.org/doc/manuals/r-release/R-exts.html), we'll need `ggplot2` here as well if it's used in the vignette:

> The ‘Suggests’ field uses the same syntax as ‘Depends’ and lists packages that are not necessarily needed. This includes packages used only in examples, tests or vignettes (see Writing package vignettes), and packages loaded in the body of functions. E.g., suppose an example11 from package foo uses a dataset from package bar. Then it is not necessary to have bar use foo unless one wants to execute all the examples/tests/vignettes: it is useful to have bar, but not necessary. Version requirements can be specified but should be checked by the code which uses the package.

Reply:
Fixed in [a3c4fde](https://github.com/MichaelChirico/geohashTools/pull/33/commits/a3c4fde3d47ee389fe3e4b5fc95fff7fe6747995)